src/pages/AdminBooksPage/AdminBooksPage.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from "@testing-library/react";
+import { MemoryRouter } from "react-router";
+import AdminBooksPage from ".";
+import userEvent from "@testing-library/user-event";
+
+describe("Test admin books page", () => {
+  test("should render new book button", async () => {
+    render(
+      <MemoryRouter>
+        <AdminBooksPage />
+      </MemoryRouter>
+    );
+
+    const newBookButton = await screen.findByTestId("new-book-button");
+
+    expect(newBookButton).toBeInTheDocument();
+  });
+});
```

Comment:
You should add one test that shows what happens when button is clicked and goes through the whole flow, fills the form, submits the book and checks if success notification shows up. 
NewBookForm.test covers only the form, we should also test the integration and the whole flow.